src/logger.ts

```diff
@@ -3,7 +3,16 @@ import Pino, { LoggerOptions, Logger } from 'pino'
 export default function DefaultLogger (options?: LoggerOptions): Logger {
   const isDeployed = process.env.NODE_ENV === 'production'
   return Pino({
-    level: isDeployed ? 'info' : 'debug',
+    level: (() => {
+      switch (process.env.NODE_ENV) {
```

Comment:
(nit) I like it. I think with this change then line #4 (`const isDeployed = process.env.NODE_ENV === 'production'`) is no longer needed

Reply:
It's still needed for the `prettyPrint`, but I just inlined it there and got rid of the line 4.